src/PhpSpreadsheet/Writer/Xlsx/Rels.php

```diff
@@ -208,13 +207,18 @@ public function writeWorksheetRelationships(\PhpOffice\PhpSpreadsheet\Worksheet\
         }
 
         if (($pWorksheet->getDrawingCollection()->count() > 0) || (count($charts) > 0) || $drawingOriginalIds) {
-            $relPath = '../drawings/drawing' . $pWorksheetId . '.xml';
-            $rId = ++$d;
+            $rId = 1;
 
+            // Use original $relPath to get original $rId.
+            // take first. In future can be overriten
+            // (! synchronize with \PhpOffice\PhpSpreadsheet\Writer\Xlsx\Worksheet::writeDrawings)
+            $relPath = array_key_first($drawingOriginalIds);
```

Comment:
This is the most important line in this Fix!
Only such $relPath can be used properly in the next two lines.

Reply:
array_key_first works only >= php 7.3 

https://www.php.net/manual/en/function.array-key-first.php

PhpSpreadsheet also supports 7.2 until Nov. 2020

Reply:
My mistake.
Fixed.
Thank you!